### PR TITLE
updated the classification for AVSD1

### PR DESCRIPTION
**Pull request by sabrinatoro**
addresses #5758

### CODE REVIEW

File: src/ontology/mondo-edit.obo [hunk 1]
```diff
@@ -230445,9 +230445,11 @@ xref: OMIM:606215 {source="MONDO:equivalentTo"}
 xref: Orphanet:98722 {source="OMIM:606215"}
 xref: UMLS:C1389016 {source="OMIM:606215", source="MONDO:notFoundInDiseaseSubset"}
 xref: UMLS:C1853513 {source="OMIM:606215", source="MONDO:ncbi_mim2gene_medline", source="MONDO:notFoundInDiseaseSubset"}
-is_a: MONDO:0003847 ! Mendelian disease
-is_a: MONDO:0020290 {source="DC-OMIM:606215", source="OMIM:606215"} ! atrioventricular septal defect
+is_a: MONDO:0020573 {source="OMIM:606215"} ! inherited disease susceptibility
+relationship: excluded_subClassOf MONDO:0020290 {source="DC-OMIM:606215", source="OMIM:606215"} ! atrioventricular septal defect
+relationship: predisposes_towards MONDO:0020290 {source="OMIM:606215"} ! atrioventricular septal defect
 property_value: confidence "0.0" xsd:double
+property_value: IAO:0000233 https://github.com/monarch-initiative/mondo/issues/5758 xsd:anyURI
```
Comment (nicolevasilevsky) [inline src/ontology/mondo-edit.obo:230452]: @sabrinatoro FYI, i added this

File: src/ontology/mondo-edit.obo [hunk 2]
```diff
@@ -230445,9 +230445,11 @@ xref: OMIM:606215 {source="MONDO:equivalentTo"}
 xref: Orphanet:98722 {source="OMIM:606215"}
 xref: UMLS:C1389016 {source="OMIM:606215", source="MONDO:notFoundInDiseaseSubset"}
 xref: UMLS:C1853513 {source="OMIM:606215", source="MONDO:ncbi_mim2gene_medline", source="MONDO:notFoundInDiseaseSubset"}
-is_a: MONDO:0003847 ! Mendelian disease
-is_a: MONDO:0020290 {source="DC-OMIM:606215", source="OMIM:606215"} ! atrioventricular septal defect
+is_a: MONDO:0020573 {source="OMIM:606215"} ! inherited disease susceptibility
+relationship: excluded_subClassOf MONDO:0020290 {source="DC-OMIM:606215", source="OMIM:606215"} ! atrioventricular septal defect
+relationship: predisposes_towards MONDO:0020290 {source="OMIM:606215"} ! atrioventricular septal defect
```
Comment (nicolevasilevsky) [inline src/ontology/mondo-edit.obo:230450]: @sabrinatoro FYI, i added this

**Review by nicolevasilevsky — APPROVED** [review]

File: src/ontology/mondo-edit.obo (hunk 2 above)
Comment (sabrinatoro) [reply src/ontology/mondo-edit.obo:230450]: thank you! I thought about it, then obviously forgot! :-)

Comment (nicolevasilevsky) [reply src/ontology/mondo-edit.obo:230450]: All good! that's why we have reviewers :)

**Pull request merged**